### ISSUE TITLE
EIDW crashes due to duplicate runway declarations

### ISSUE THREAD

**Issue by Wittner** [openscope/openscope#415]
```
Fms.js:447 Uncaught TypeError: Expected instance of RunwayModel, but received null
    at Fms.value (Fms.js:447)
    at Fms.value (Fms.js:1303)
    at Fms.value (Fms.js:310)
    at new Fms (Fms.js:191)
    at AircraftInstanceModel.value (AircraftInstanceModel.js:294)
    at new AircraftInstanceModel (AircraftInstanceModel.js:181)
    at AircraftController.value (AircraftController.js:164)
    at AircraftController.createAircraftWithSpawnPatternModel (AircraftController.js:117)
    at SpawnScheduler.value (SpawnScheduler.js:96)
    at new SpawnScheduler (SpawnScheduler.js:49)

App.js:346 Uncaught TypeError: Cannot read property 'canvas_complete' of null
    at App.value (App.js:346)
    at App.value (App.js:378)
    at App.js:436
```
Nate suggested replacing LocalStorage airport value with KSFO and rebooting which got the app working.

**Comment by n8rzz** [comment]
this airport file contains duplicate definitions for `arrivalRunway` and `departureRunway`: 
```javascript
    "arrivalRunway": "28",
    "departureRunway": "28",
    "arrivalRunway": "22",
    "departureRunway": "22",
```
My guess is it's sticking with the last definition for runway `22` which is not defined in the airport file.

**Comment by erikquinn** [comment]
Will be resolved by #209.

Rolled into #208. Closing.
